python/lsst/ap/verify/testUtils.py

```diff
@@ -0,0 +1,62 @@
+#
+# This file is part of ap_verify.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (http://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+"""Common code for ap_verify unit tests.
+"""
+
+import unittest
+
+import lsst.utils.tests
+
+import lsst.pex.exceptions as pexExcept
+from lsst.ap.verify.config import Config
+
+
+class DataTestCase(lsst.utils.tests.TestCase):
```

Comment:
Are you sure? We used this pattern a lot in `afw` and didn't run into any problems. The fact that it's in `python/` (necessary to ensure it's on the path) should make it immune to that kind of bug.

Reply:
I assume the test classes themselves inherit from `DataTestCase` so I don't think there is a problem. The problem is when you put a class in a test file that looks like a class of tests but isn't really.

Reply:
I don't agree that it's a mixin (though maybe that word means something different in Python than in Java); it's a specific type of test case rather than extra functionality.

Anyway, I propose we revisit this later (and maybe the aforementioned `afw` test utilities at the same time), since this bit of code sharing is not relevant to the actual speed-up work.

Reply:
Ok. Maybe I should make a Community post to discuss it?

Reply:
Maybe discuss with Russell first, because `lsst.afw.geom.testUtils.TransformTestBaseClass` (the "lots" of uses in `afw` I thought I remembered :cold_sweat:) was his idea.